src/colvarcomp.h

```diff
@@ -237,6 +237,15 @@ class colvar::cvc
     x = new_value;
   }
 
+  /// Set frequency at which the list of contributing atoms will be updated
+  virtual int set_atom_list_frequency(int new_frequency);
+
+  /// If needed, update the requested list of atoms from this group
+  virtual int update_requested_atoms(cvm::atom_group *dyn_atoms);
```

Comment:
I will need to implement this for `cvm::atom_group_soa` in https://github.com/Colvars/colvars/pull/788.